test/src/androidTest/java/com/quickbirdstudios/test/FullSurveyTest.kt

```diff
@@ -125,15 +117,17 @@ internal class FullSurveyTest : PageTest {
                         Assert.assertTrue(questionResult.answer.toString() == NumberStepInput)
                     is MultipleChoiceQuestionResult -> {
                         val expectedResult = listOf(
-                            (activityRule
-                                .activity
-                                .multipleChoiceStep
-                                .answerFormat as MultipleChoiceAnswerFormat
+                            (
+                                activityRule
+                                    .activity
+                                    .multipleChoiceStep
+                                    .answerFormat as MultipleChoiceAnswerFormat
                                 ).textChoices[0],
-                            (activityRule
-                                .activity
-                                .multipleChoiceStep
-                                .answerFormat as MultipleChoiceAnswerFormat
+                            (
```

Comment:
same

Reply:
When i format it via klint it's seems like that.